src/command/build.rs

```diff
@@ -239,7 +246,7 @@ impl Build {
         Ok(())
     }
 
-    fn get_process_steps(mode: InstallMode) -> Vec<(&'static str, BuildStep)> {
+    fn get_process_steps(mode: InstallMode, no_pack: bool) -> Vec<(&'static str, BuildStep)> {
```

Comment:
```suggestion
    fn get_process_steps(&self) -> Vec<(&'static str, BuildStep)> {
```
Would give it access to both without having to pass any parameters at all.